src/util/transaction.js

```diff
@@ -52,81 +61,207 @@ export const TRANSITIONS = [
   TRANSITION_REVIEW_2_BY_PROVIDER,
 ];
 
-const txLastTransition = tx => ensureTransaction(tx).attributes.lastTransition;
+/**
+ * Actors
+ *
+ * There are 4 different actors that might initiate transitions:
+ */
+
+// Roles of actors that perform transaction transitions
+export const TX_TRANSITION_ACTOR_CUSTOMER = 'customer';
+export const TX_TRANSITION_ACTOR_PROVIDER = 'provider';
+export const TX_TRANSITION_ACTOR_SYSTEM = 'system';
+export const TX_TRANSITION_ACTOR_OPERATOR = 'operator';
+
+export const TX_TRANSITION_ACTORS = [
+  TX_TRANSITION_ACTOR_CUSTOMER,
+  TX_TRANSITION_ACTOR_PROVIDER,
+  TX_TRANSITION_ACTOR_SYSTEM,
+  TX_TRANSITION_ACTOR_OPERATOR,
+];
 
-export const txIsEnquired = tx => txLastTransition(tx) === TRANSITION_ENQUIRE;
+/**
+ * States
+ *
+ * You should not use these constants outside of this file.
+ */
+const STATE_INITIAL = 'initial';
+const STATE_ENQUIRY = 'enquiry';
+const STATE_PREAUTHORIZED = 'preauthorized';
+const STATE_DECLINED = 'declined';
+const STATE_ACCEPTED = 'accepted';
+const STATE_CANCELED = 'canceled';
+const STATE_DELIVERED = 'delivered';
+const STATE_REVIEWED = 'reviewed';
+const STATE_REVIEWED_BY_CUSTOMER = 'reviewed-by-customer';
+const STATE_REVIEWED_BY_PROVIDER = 'reviewed-by-provider';
 
-export const transitionsToRequested = [TRANSITION_REQUEST, TRANSITION_REQUEST_AFTER_ENQUIRY];
-export const txIsRequested = tx => transitionsToRequested.includes(txLastTransition(tx));
+/**
+ * Description of transaction process
+ *
+ * You should keep this in sync with transaction process defined Flex API
```

Comment:
"defined in Marketplace API"

Flex API should be called Marketplace API like in [Flex concepts](https://www.sharetribe.com/docs/background/concepts/)